socialhome/streams/app/main.js

```diff
@@ -1,6 +1,6 @@
 import Vue from "vue"
-import {VueMasonryPlugin} from "vue-masonry"
 import BootstrapVue from "bootstrap-vue"
+import VueMasonryPlugin from "vue-masonry"
```

Comment:
Hmm.. How come this started working, when I added the braces I did it because this didn't work. Also, the main readme of the library has braces: https://github.com/shershen08/vue-masonry